### PR TITLE
Google basemaps thumbnails

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-06-22 at 18 39 32

Thanks to @mariodelvalle.

@saleiva @javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4481/))

**Comment by xavijam** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4483/))

**Comment by xavijam** [comment]
@javierarce, could you take a look mate, buddy, fella... please?

**Comment by javierarce** [comment]
Looks :tada: to me.

**Comment by xavijam** [comment]
Sorry @javierarce, but we should check it again :(, also @javisantana.

**Comment by xavijam** [comment]
And also, I've checked that this is getting more complicated, we should "redone" this component when we have time.

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js [hunk 1]
```diff
@@ -210,6 +210,9 @@ cdb.admin.DropdownBasemap = cdb.ui.common.Dropdown.extend({
     var className = this._getCleanClassName(className);
     if (className === 'googlemaps') {
       className = subview.model.get('base_type');
+      // Adding the custom base name to set properly
+      // the background image
+      className += subview.model.get('base_name') ? ( '.' + subview.model.get('base_name') ) : '';
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dropdown/basemap_dropdown.js]: className has to be always 'googlemaps', due to some checks we have. So we need to add a new "attribute" (```base_name```) in order to know the type of the basemap and set correctly the thumbnail.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4498/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4500/))

**Comment by xavijam** [comment]
bump! :)

**Comment by xavijam** [comment]
Tested in staging, working with google and no google accounts. So merging!

**Pull request merged**